packages/temen/src/isArguments/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * value가 arguments인지 확인
+ *
+ * @param {*} value 검사할 값
+ * @returns {boolean} arguments이면 true 반환 아닌 경우 false 반환
+ * @example
+ *
+ * ```js
+ * isArguments(function() { return arguments }())
+ * // => true
+ *
+ * isArguments([1, 2, 3])
+ * // => false
+ * ```
+ */
+
+function isArguments(value: any): boolean {
+  return toString.call(value) === '[object Arguments]';
```

Comment:
`_internal/getType` 함수 사용해주시면 좋을 것 같아유!

```suggestion
  return getType(value) === '[object Arguments]';
```

Reply:
https://github.com/Lubycon/lubycon-frontend-libraries/pull/122/commits/2033f768377f507260b6f1ada2239389bebf5745

반영했숩니당